### PR TITLE
update to use psr-11

### PR DESCRIPTION
**Pull request by harikt**
Hi @elazar , 

I was looking at Auryn and came across https://github.com/rdlowrey/auryn/issues/77 and https://github.com/container-interop/container-interop/issues/75 .

As psr-11 is already there may be we could update the library to just depend on psr/container. Even though without this PR it is satisfied via container-interop . May be we can bump this up ?

Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ integrating Auryn with third-party libraries that use *container-interop*, such
 as [zend-expressive](https://github.com/zendframework/zend-expressive).
 
 The implementation in this repository takes a small amount of liberty with this
-passage from [Section 1.1](https://github.com/container-interop/container-interop/blob/master/docs/ContainerInterface.md#11-basics)
+passage from [Section 1.1](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-11-container.md#11-basics)
```
Comment (harikt) [inline README.md]: Hi @elazar , 

I think probably we can get rid of this section, for the current get and has doesn't have the problem mentioned as in container-interop. Let me know what you think.

Comment (elazar) [reply README.md]: @harikt I'm not sure I follow. The same quote exists in both documents. Can you elaborate?

Comment (harikt) [reply README.md]: Hi @elazar , 

The delegate lookup feature is not currently in `PSR-11` , but is only present in `container-interop`. 

See discussion over https://github.com/laravel/internals/issues/803#issue-259409277 .

Let me know if that helps! .

Thank you.

Comment (elazar) [reply README.md]: @harikt I think I see what you're getting at. I'm open to suggestions for amending or removing the cited passage.

**Comment by shadowhand** [comment]
I think using something like [northwoods/container](https://github.com/northwoods/container) is more appropriate, as it wraps Auryn rather than extending it. This is more consistent with SOLID principles.

**Comment by elazar** [comment]
@shadowhand Seems sort of similar to elazar/auryn-configuration ?

**Comment by shadowhand** [comment]
@elazar it certainly has some crossover. And also with https://packagist.org/packages/equip/config.

**Comment by elazar** [comment]
Closing in favor of #4.

**Pull request closed without merging**